azure-pipelines-data-container.yml

```diff
@@ -0,0 +1,34 @@
+trigger:
+  - master
+  - datacontainerpipelinereview
+
+schedules:
+- cron: "0 0 * * *"
+  displayName: Nightly build
+  branches:
+    include:
+    - master
+  always: true
+
+pool:
+  vmImage: 'Ubuntu-16.04'
+
+variables:
+  IMAGE_TAG_PREFIX: $[format('1.0.{0:HHm}', pipeline.startTime)]
+
+steps:
+- task: Docker@2
+  displayName: Login to ACR
+  inputs:
+    command: login
+    containerRegistry: $(CONTAINER_REGISTRY_NAME)
+- task: Docker@2
+  displayName: Build and Push
+  inputs:
+    command: buildAndPush
+    containerRegistry: $(CONTAINER_REGISTRY_NAME)
+    repository: $(REPOSITORY_NAME_PREFIX)-$(Build.SourceBranchName)
+    Dockerfile: scripts/datacontainer/Dockerfile
+    tags: |
+      latest
+      $(IMAGE_TAG_PREFIX)$(Build.BuildNumber)
```

Comment:
Maybe you need some kind of cleanup tool here every month?

Reply:
yes. https://docs.microsoft.com/en-us/azure/container-registry/container-registry-auto-purge#run-in-a-scheduled-task